packages/projects-docs/pages/learn/repositories/review.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Code Reviews
+description: Learn how to improve the process of code reviews in CodeSandbox.
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Using CodeSandbox for code reviews
+
+<Callout>
+Code Reviews are currently only available for GitHub repositories.
+</Callout>
+
+CodeSandbox is the perfect place to conduct code reviews. By opening the branch from the GitHub PR in CodeSandbox, you will see the code changes, code preview and conversations all in one place. 
+
+Reviews on CodeSandbox are synced with GitHub. It doesn't matter if you start a review in GitHub and finish it in CodeSandbox–all comments will be reflected in both GitHub and CodeSandbox.
+
+## Getting started
+Reviews require a repository admin to install the [CodeSandbox GitHub App](/learn/integrations/github-app). Once that is installed, you will see a comment in your GitHub PRs with links to open the branch and the preview.
+
+## How it works
+Once a PR is created, you will see links to the CodeSandbox branch. If you open the branch in the web editor, you will see something like this:
+
+![CodeSandbox Review](../images/review-start.png)
+
+### Reviewers
+All the of the requested reviewers and their review status are listed at the top of the PR tab. If they have left a message with their review, you can view it by clicking the chevron to the left of their name.
+
+### Changes by file
+This section lists all the changes that were made to the code in the PR. 
+At this time, comments are only shown on diff files.
+
+### Comments
+You can add a comment by opening a diff file and hovering your mouse between the line numbers to see a plus button, as highlighted below.
```

Comment:
Maybe we should say "a plus icon on the left of the line number", because I was confused for a minute with the other plus icon at the right of the line number.

Reply:
that makes sense

Reply:
okay, I changed the language and added a cursor to the image